src/Styles/index.css

```diff
@@ -3,6 +3,7 @@
     height: 80px;
```

Comment:
More often than not, you shouldn't strictly set the height of the content. It is better to limit the maximum or minimum.